src/atoms/slider/slider.styles.ts

```diff
@@ -0,0 +1,117 @@
+import { css } from 'styled-components';
+import { Styled } from '../../theme';
+import { SliderProps } from './types';
+
+const track = css<SliderProps>`
+  box-sizing: border-box;
+  border: none;
+  height: 8px;
+  background: ${(props) => (props.value === 100 ? props.theme.textShades.SHADE_MINUS_3 : props.theme.containerAndCardShades.NEUTRAL_SHADE_0)};
+  border-radius: 12px;
+`;
+
+const trackFill = css<SliderProps>`
+  ${track};
+  height: 8px;
+  background-color: ${(props) => (props.value === 100 ? props.theme.textShades.SHADE_MINUS_3 : props.theme.containerAndCardShades.NEUTRAL_SHADE_0)};
+  &:hover {
+    background-color: ${(props) => props.theme.colors.primary.DARK_BLUE} ;
+  }
+`;
+
+const fill = css`
+  height: 8px;
+  background: ${(props) => props.theme.colors.primary.WATER_BLUE};
+  border-radius: 12px;
+`;
+
+const thumb = css<SliderProps>`
+  box-sizing: border-box;
+  border: none;
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  background:  ${(props) => (props.value === 100 ? props.theme.colors.system.WHITE : props.value === 0 ? props.theme.containerAndCardShades.NEUTRAL_SHADE_0 : props.theme.colors.primary.WATER_BLUE)};
+  &:hover {
+    background: ${(props) => props.theme.colors.system.WHITE}
+  };
+`;
+
+export const Input = Styled.input<SliderProps>`
+  cursor: pointer;
+  &,
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  margin: 0;
+  padding: 0;
+  height: 20px;
+  background: transparent;
+
+
+  &::-webkit-slider-runnable-track {
+    ${trackFill};
+  }
+
+  &::-moz-range-track {
+    ${track};
+  }
+
+  &::-ms-track {
```

Comment:
@shan8851 @ArinCantCode are we supporting ms ?, and i 10000% sure that npm build will use autoprefixer no ?

Reply:
@goshatravin What do you mean by supporting? Frontend for sure already uses `-ms`

Reply:
@ArinCantCode are we supporting Internet Explorer  ?

Reply:
@goshatravin internet explorer doesn't support metamask but edge does